src/render/program.js

```diff
@@ -90,7 +90,6 @@ class Program<Us: UniformBindings> {
             }
         }
 
-        this.binderUniforms = configuration.getUniformBindings(context);
```

Comment:
This should still live on `Program`, so that `set`s are deduped across different `ProgramConfiguration`s.